nav2_msgs/action/Spin.action

```diff
@@ -1,9 +1,18 @@
+# Error codes
+# Note: The expected priority order of the error should match the message order
+uint16 NONE=0
+uint16 UNKNOWN=700
+uint16 TIMEOUT=701
+uint16 TF_ERROR=702
+uint16 COLLISION_AHEAD=703
```

Comment:
No invalid input?

Reply:
Spin currently has no check to invalidate any input.